rest_framework/fields.py

```diff
@@ -53,14 +53,30 @@ def is_simple_callable(obj):
     """
     True if the object is a callable that takes no arguments.
     """
+    if not hasattr(inspect, 'signature'):
+        return py2k_is_simple_callable(obj)
```

Comment:
I'd be very happy to do so yup. Let's defer brining it in til 3.5 tho' (Not too far off right now)